experimental/handoff-library/csharp_dotnetcore/samples/LivePersonProxyAssistant/LivePersonConnector/ConversationMap.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Concurrent;
+using Microsoft.Bot.Schema;
+
+namespace LivePersonConnector
```

Comment:
Can you make any necessary changes in the existing LivePersonConnector instead of cloning it?